R/git_help.R

```diff
@@ -49,7 +49,7 @@ get_datetime <- function(commit_val) {
 get_msg <- function(commit_val) {
   stopifnot(git2r::is_commit(commit_val))
   
-  base::substr(commit_val@summary, start = 1, stop = 15)  
+  base::substr(commit_val[[4]], start = 1, stop = 15)  
```

Comment:
hi @EngineerDanny this is OK but would be better if you could use a named reference ( $summary ?) rather than a number, [[4]], is that possible ?